kernel/configs/x86_64_kata_kvm_4.19.x

```diff
@@ -2597,7 +2597,9 @@ CONFIG_FANOTIFY=y
 # CONFIG_QUOTA is not set
 CONFIG_AUTOFS4_FS=y
 CONFIG_AUTOFS_FS=y
-# CONFIG_FUSE_FS is not set
+CONFIG_FUSE_FS=y
+# CONFIG_CUSE is not set
+CONFIG_VIRTIO_FS=y
```

Comment:
As these two `CONFIG` items are only present with the patches in this PR (that is, they are not upstream yet afaict), we can't apply them to the fragments PR until this PR has landed.
They look find in the context of this PR though.